app/src/components/Leaderboard.js

```diff
@@ -59,7 +69,7 @@ class Leaderboard extends React.Component {
         this.state.players.map((player, index) =>
```

Comment:
Since we're calculating the player's index based on their count of wins, we can remove index from this line

```javascript
this.state.players.map((player, index) =>
```

and this line
```javascript
<div className="row Robo-Font grid-body" key={player.name + "-player" + index}>
```

Reply:
great point! just removed those :)